ci-operator/config/stolostron/search-v2-api/stolostron-search-v2-api-main.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.1=7-linux:
```

Comment:
There is an extraneous equal sign in this line `go1.1=7`

Reply:
thanks!